<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hey! Healer - 메인</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="q_heyhealer_loginout_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>420</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="q_login_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hey! Healer 에 오신걸 환영합니다!</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_user_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>관리자 님</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="q_heyhealer_loginout_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>5</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QWidget" name="q_user_buttons" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <widget class="QPushButton" name="q_heydealer_login_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>헤이딜러 로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="q_heydealer_login_button_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>입찰 필터</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="q_heydealer_login_button_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>감가 설정</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>q_heyhealer_loginout_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
